dev/cnf/build.gradle

```diff
@@ -30,9 +30,9 @@ dependencies {
     binaries "org.jsoup:jsoup:1.7.2"
     binaries "com.ibm.ws.javax.j2ee:servlet:3.1"
     binaries "org.jmockit:jmockit:1.25"
-    binaries "biz.aQute.bnd:biz.aQute.bnd:3.5.0"
-    binaries "biz.aQute.bnd:biz.aQute.bnd.annotation:3.5.0"
-    binaries "biz.aQute.bnd:org.osgi.impl.bundle.repoindex.cli:3.5.0"
+    binaries "biz.aQute.bnd:biz.aQute.bnd:4.0.0"
+    binaries "biz.aQute.bnd:biz.aQute.bnd.annotation:4.0.0"
+    binaries "biz.aQute.bnd:biz.aQute.bnd.gradle:4.0.0"
```

Comment:
I don't think you need the gradle plugin jar in here. The buildscript already provides the dependency on gradle and access to the bnd gradle classes.